neo/SmartContract/VerificationControl.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Neo.VM;
+using System.Collections.Generic;
+
+namespace Neo.SmartContract
+{
+    public class VerificationControl : GasControl
+    {
+        private readonly static List<OpCode> _bannedOpcodes = new List<OpCode>(new OpCode[] { /*OpCode.NOP*/ });
```

Comment:
Its in a good direction shargon, however needs a little more complication here. To ban backwards jumps we need to analyse the code and its first parameter... so, a callback returning true/false is better than a static list.
I've been thinking and another option would be to have different JMPF that can only jump forward. If we do this, its easier to ban. 3 more: JMPF, JMPFIF and JMPFIFNOT . A callback can avoid this and give more control on used opcodes.

Reply:
This gonna be an interesting thing!

Reply:
Do you want to handle these bannedOpcodes in this PR or split in two?

Reply:
@igormcoelho